integration_tests/sdk/resources_test.py

```diff
@@ -110,3 +110,58 @@ def fn_expect_failure():
         engine=engine,
         expect_success=False,
     )
+
```

Comment:
Can we add a test for a typo in the `gpu_resource_name` and check that the workflow fails? I guess it would need to be rebased onto https://github.com/aqueducthq/aqueduct/pull/720 right, since right now it would hang. Maybe it could be a separate PR?

Reply:
Yes, I'll tack on a separate PR for this on top of 720.

Reply:
https://linear.app/aqueducthq/issue/ENG-2014/add-sdk-integration-test-for-gpu-resource-name-mismatch-unavailable